assets/js/app/DataLayer.js

```diff
@@ -112,12 +112,13 @@ LocusZoom.PositionsDataLayer = function(){
 
     LocusZoom.DataLayer.apply(this, arguments);  
     this.id = "positions";
-    this.fields = ["id","position","pvalue","refAllele","ld:state"];
+    this.xfield = "position";
+    this.yfield = "pvalue|neglog10";
+    this.fields = ["id", this.xfield, this.yfield, "refAllele", "ld:state"];
 
     this.postget = function(){
         this.data.map(function(d, i){
             this.data[i].ld = +d["ld:state"];
```

Comment:
Does this just do a numeric conversion? Was there a time that ld wasn't numeric? We could have a "|numeric" transformation but I think this might not be necessary at all. Thoughts?